homeassistant/components/teslemetry/manifest.json

```diff
@@ -7,5 +7,5 @@
   "iot_class": "cloud_polling",
   "loggers": ["tesla-fleet-api"],
   "quality_scale": "platinum",
-  "requirements": ["tesla-fleet-api==0.6.2"]
+  "requirements": ["tesla-fleet-api==0.7.2"]
```

Comment:
Can we do this bump in a preliminary PR first so this PR only is for `tesla_fleet`?

Reply:
https://github.com/home-assistant/core/pull/122073

Reply:
Please rebase now after bump is merged 👍

Reply:
Done